app/src/main/java/com/datastax/themis/cli/commands/AbstractCommand.java

```diff
@@ -0,0 +1,21 @@
+package com.datastax.themis.cli.commands;
+
+import picocli.CommandLine;
+
+public abstract class AbstractCommand {
```

Comment:
We already had a good set of params in common for each op so the addition of ks/table params made the case for consolidating all these things into an abstract superclass pretty obvious.